baseapp-auth/baseapp_auth/rest_framework/jwt/serializers.py

```diff
@@ -26,13 +26,12 @@ def get_claim_serializer_class(cls):
 class BaseJwtLoginSerializer(
     CustomClaimSerializerMixin, LoginPasswordExpirationMixin, TokenObtainPairSerializer
 ):
-    @classmethod
-    def get_token(cls, user):
+    def get_token(self, user):
         token = super().get_token(user)
 
         # Add custom claims
-        if cls._claim_serializer_class:
-            data = cls.get_claim_serializer_class()(user).data
+        if self._claim_serializer_class:
+            data = self.get_claim_serializer_class()(user, context=self.context).data
```

Comment:
This allows relative urls to be converted to absolute urls when serializing the profile image